client/src/components/add-new-trip/NewTripInputs.jsx

```diff
@@ -21,8 +21,15 @@ export default function NewTripInputs({
   const [startDateValue, setStartDateValue] = useState(new Date());
   const [endDateValue, setEndDateValue] = useState(new Date());
   const [tripName, setTripName] = useState(tripDestination);
+  const [textFieldLabel, setTextFieldLabel] = useState("Trip Location");
+  const [textFieldError, setTextFieldError] = useState(false);
 
   const startNewTrip = async () => {
+    if (tripName.length <= 2) {
```

Comment:
What if they type in LA and not Los Angeles, I think we should accept that. 

Since Itay is making a call to google location api anyway, LA might work. @itayneu can you confirm that? 

We should just make sure it's more than 1 letter I guess

Reply:
LA would work, it was one of my tests. I would limit to 2+ letters

Reply:
updated to < 2 , please check/merge